src/main/java/ch/presentium/backend/api/student/request/StudentRequest.java

```diff
@@ -0,0 +1,18 @@
+package ch.presentium.backend.api.student.request;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+@Data
+@Builder
+@Jacksonized
+public class StudentRequest {
+
+    @NotNull private final String firstName;
```

Comment:
```suggestion
    private final @NotBlank String firstName;
```

pareil pour les autres strings, ça évitera des strings vide ou avec que des espaces